packages/framework/src/Model/Product/ProductOnCurrentDomainFacade.php

```diff
@@ -278,9 +278,9 @@ public function getProductFilterCountDataForSearch(
     /**
      * @return array
      */
-    public function getAllListableProducts(): array
+    public function getAllOfferedProducts(): array
```

Comment:
What is the reason for renaming this method? Is that so important for us to cahnge the interface? I guess is is beceause `Offered` should return all products and `Listable` only return some chunk of product dependent on pagination?

It is a bit confusing to me but I belive we will not want to return all products in one request. Also this is not the scope of this PR.